test/dlc_tests/conftest.py

```diff
@@ -908,6 +1122,9 @@ def pytest_configure(config):
         "markers", "processor(cpu/gpu/eia/hpu): explicitly mark which processor is used"
     )
     config.addinivalue_line("markers", "efa(): explicitly mark to run efa tests")
+    config.addinivalue_line(
+        "markers", "allow_p4de_use(): explicitly mark to allow test to use p4de instance types"
```

Comment:
why is this needed? is it to conserve on p4de usage?

Reply:
That's right. If a test does not have `allow_p4de_use` as one of its markers, it will be skipped when the instance type environment variable on the test job is defined as `p4de.24xlarge`.